pypeclub/OpenPype
# Maya: Remove hardcoded requirement for maya/ start for image file prefix

## Brief description

This PR is to resolve #3872 

This is just a code prototype - I haven't tested this. If someone could, that'd be very appreciated!

## Description

This will remove the `maya/` requirement from the file prefix and by default moves that behavior into the `workspace.mel` by adding it into the "images" file rule defining where Maya puts its renders.

Moving it there allowed less logic to be implemented, and also allowing the customization as described in #3872. To NOT render into a `maya/` subfolder one would only need to set the "images" file rule to `renders` instead of `renders/maya`

## Testing notes:

1. Launch Maya
2. Submit renders
3. Ensure renders end up in similar position as before - that is `renders/maya/{scene}/{renderlayer}_{renderpass}` or alike.
